doc/classes/TextureButton.xml

```diff
@@ -29,18 +29,21 @@
 		</member>
 		<member name="texture_disabled" type="Texture2D" setter="set_texture_disabled" getter="get_texture_disabled">
 			Texture to display when the node is disabled. See [member BaseButton.disabled].
+			If not assigned, when the [TextureButton] is disabled it will display [member texture_normal] (or nothing, if [member texture_normal] itself has no texture assigned).
```

Comment:
```suggestion
			Texture to display when the node is disabled. See [member BaseButton.disabled]. If not assigned, displays [member texture_normal] instead.
```

Reply:
When `texture_disabled` and `texture_normal` are both assigned, then `texture_disabled` is used:

https://github.com/user-attachments/assets/9ca36001-ce8f-4801-b4f5-46c3a2f567fc

When `texture_disabled` is not assigned, but `texture_normal` is assigned, then `texture_normal` is used:

https://github.com/user-attachments/assets/8d6ff94e-5219-4121-b287-6d92fa879918

When neither `texture_disabled` nor `texture_normal` are assigned, then no texture is used:

https://github.com/user-attachments/assets/cb6afd66-1c3e-4ded-8466-dd3a42a214f5

Reply:
```suggestion
			Texture to display when the node is disabled. See [member BaseButton.disabled]. If not assigned, the [TextureButton] displays [member texture_normal] instead.
```